Scripts/Source/UD_TrapManifest_ME_Script.psc

```diff
@@ -0,0 +1,99 @@
+Scriptname UD_TrapManifest_ME_Script extends activemagiceffect  
+
+import UnforgivingDevicesMain
+;import UD_CustomDevice_RenderScript 
+
+;Int Property rare_device_chance = 25 auto
+UD_AbadonQuest_script Property AbadonQuest auto
+UD_libs Property UDlibs auto
+UDCustomDeviceMain Property UDCDmain auto
+zadlibs Property libs auto
+
+Event OnEffectStart(Actor akTarget, Actor akCaster)
+
+    if !akTarget.wornhaskeyword(libs.zad_deviousheavybondage)
+        UDCDmain.DisableActor(akTarget)
+
+		bool loc_haveSuit = akTarget.wornhaskeyword(libs.zad_deviousSuit)
+		Formlist loc_formlist = none
+	    Armor device = none
+        Int loc_arousal = UDCDmain.UDOM.getArousal(akTarget)
+
+;        if Utility.randomInt(1,99) < Round(UDCDMain.UD_BlackGooRareDeviceChance*fRange(loc_arousal/50,1.0,2.0))
```

Comment:
Remove long comment block

Reply:
Ah, my blasted habit of avoiding removing old code and preserve it in comments...
Will clean it up.

Reply:
Cleaned it up, should be good now.